src/views/RecIM/views/Admin/index.jsx

```diff
@@ -273,6 +347,188 @@ const Admin = () => {
+            <Card elevation={10}>
+              <CardHeader
+                title={
+                  <>
+                    <Typography className={styles.title}>Rec-IM Admin Report</Typography>
+                  </>
+                }
+              />
+              <CardContent>
+                <Card>
+                  <CardContent>
+                    <Typography className={styles.reportSubtitle}>
+                      {'From: ' + (recimReport && new Date(recimReport.StartTime).toLocaleString())}
+                    </Typography>
+                    <Typography className={styles.reportSubtitle}>
+                      {'To: ' + (recimReport && new Date(recimReport.EndTime).toLocaleString())}
+                    </Typography>
+                  </CardContent>
+                </Card>
+                <Card className={styles.reportCard}>
+                  <CardHeader
+                    className={styles.cardHeader}
+                    title={
+                      <Typography className={styles.cardHeader}>
+                        Active Participants:{' '}
+                        {' ' + (recimReport && recimReport.NumberOfActiveParticipants)}
+                      </Typography>
+                    }
+                  />
+                  <CardContent>
+                    {recimReport &&
+                      recimReport.ActiveParticipants.map((participants) => (
+                        <>
+                          <Grid container>
+                            <Grid xl={8}>
+                              <Typography className={styles.reportText}>
+                                {'Name: ' + participants.Username}
+                              </Typography>
+                            </Grid>{' '}
+                            <Grid xl={4}>
+                              <Typography className={styles.reportText}>
+                                {' Gender: ' + participants.SpecifiedGender}
+                              </Typography>
+                              {
+                                //Combining the two typography texts fixes the print not having a space, if no other solution can be found.
+                              }
+                            </Grid>
+                          </Grid>
+                        </>
+                      ))}
+                  </CardContent>
+                </Card>
+                <Card className={styles.reportCard}>
+                  <CardHeader
+                    className={styles.cardHeader}
+                    title={
+                      <Typography className={styles.cardHeader}>
+                        New Participants:{' '}
+                        {' ' + (recimReport && recimReport.NumberOfNewParticipants)}
+                      </Typography>
+                    }
+                  />
+                  <CardContent>
+                    {recimReport &&
+                      recimReport.NewParticipants.map((participants) => (
```

Comment:
What happens if there are no `NewParticipants`?

Reply:
It actually works totally fine, I tested it to be sure, but I think since the JSON still has the NewParticipants array, it's just empty, it doesn't cause an issue.

Reply:
Great, I just wanted to check.